lively.ide/world.js

```diff
@@ -713,7 +713,7 @@ export class LivelyWorld extends World {
       if (typeof opts.customize === 'function') { opts.customize(promptMorph); }
 
       if (opts.animated) {
-        if (this.previousPrompt && this.previousPrompt.world()) {
+        if (this.previousPrompt && this.previousPrompt.world() && this.previousPrompt.transitionTo) {
```

Comment:
I guess that's fine, but prompt transitioning is broken anyways.
So let's write a note to fix that at some point in the future.

Reply:
Alright, I'll put it on the list.